apps/storefront/src/hooks/dom/utils.test.ts

```diff
@@ -756,5 +756,50 @@ describe('addProductFromProductPageToQuote', () => {
       expect(globalSnackbar.success).toHaveBeenCalled();
       expect(addQuoteDraftProduce).toHaveBeenCalled();
     });
+
+    it('correctly decodes HTML entities in SKU when matching variants', async () => {
```

Comment:
the decoding is an implementation detail, this is more like handling SKUs with especial characters in them.